ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/serde/protobuf/ValueSpecProtobufSerdeSupplier.java

```diff
@@ -15,24 +15,50 @@
 
 package io.confluent.ksql.test.serde.protobuf;
 
+import static io.confluent.connect.protobuf.ProtobufDataConfig.OPTIONAL_FOR_NULLABLES_CONFIG;
+import static io.confluent.connect.protobuf.ProtobufDataConfig.WRAPPER_FOR_NULLABLES_CONFIG;
+import static io.confluent.connect.protobuf.ProtobufDataConfig.WRAPPER_FOR_RAW_PRIMITIVES_CONFIG;
+
+import com.google.common.collect.ImmutableMap;
 import io.confluent.connect.protobuf.ProtobufConverter;
+import io.confluent.connect.protobuf.ProtobufDataConfig;
+import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient;
 import io.confluent.kafka.schemaregistry.protobuf.ProtobufSchema;
+import io.confluent.kafka.serializers.AbstractKafkaSchemaSerDeConfig;
 import io.confluent.ksql.serde.protobuf.ProtobufProperties;
 import io.confluent.ksql.serde.protobuf.ProtobufSchemaTranslator;
 import io.confluent.ksql.test.serde.ConnectSerdeSupplier;
 import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.storage.Converter;
 
 public class ValueSpecProtobufSerdeSupplier extends ConnectSerdeSupplier<ProtobufSchema> {
 
   private final ProtobufSchemaTranslator schemaTranslator;
 
+  private final ImmutableMap<String, Boolean> converterConfig;
+
   public ValueSpecProtobufSerdeSupplier(final ProtobufProperties protobufProperties) {
     super(ProtobufConverter::new);
     this.schemaTranslator = new ProtobufSchemaTranslator(protobufProperties);
+    this.converterConfig = ImmutableMap.of(
+        OPTIONAL_FOR_NULLABLES_CONFIG, protobufProperties.isNullableAsOptional(),
+        WRAPPER_FOR_NULLABLES_CONFIG, protobufProperties.isNullableAsWrapper()
+    );
   }
 
   @Override
   protected Schema fromParsedSchema(final ProtobufSchema schema) {
     return schemaTranslator.toConnectSchema(schema);
   }
+
+  protected void configureConverter(final Converter c, final boolean isKey) {
```

Comment:
nit: mark this method as `@Override`, to avoid confusion about where it's used.

Reply:
Done